tests/ext/sandbox/hook_method/hook_extended_internal_class.phpt

```diff
@@ -0,0 +1,58 @@
+--TEST--
+Extended internal classes are hookable
+--FILE--
+<?php
+
+DDTrace\hook_method('Countable', 'count', function () {
+    echo " Countable";
+});
+
+DDTrace\hook_method('ArrayObject', 'count', function () {
+    echo " ArrayObject";
+});
+
+// Runtime resolution
+if (true) {
+    class A extends ArrayObject {
+    }
+}
+
+echo "Normal:";
+(new ArrayObject([1]))->count();
+echo "\nExtended:";
+(new A)->count();
+//echo "\nAnonymous:";
+//(new class extends ArrayObject {})->count();
+
+dd_untrace('count', 'Countable');
+dd_untrace('count', 'ArrayObject');
+
+echo "\nNormal: Nohook";
+(new ArrayObject([1]))->count();
+echo "\nExtended: Nohook";
+(new A)->count();
+//echo "\nAnonymous: Nohook";
+//(new class extends ArrayObject {})->count();
+
+$anon = new class extends ArrayObject {};
```

Comment:
Just to check: this line doesn't matter at all, right? Was just missed when commenting out anon stuff?

Reply:
Yes (not quite missed, but, yes, it does not matter).